client-linuxapp/src/main.rs

```diff
@@ -425,10 +445,15 @@ async fn main() -> Result<()> {
         return Ok(());
     }
 
-    // Get owner info
-    let to1d = get_to1d_from_rv(dc.as_ref())
+    // Get rv entries
+    let rv_info = get_rv_info(dc.as_ref());
+
+    let client_list = get_client_list(rv_info.unwrap())
         .await
-        .context("Error getting to1d from rendezvous server")?;
+        .context("Error getting usable rendezvous client")?;
+
+    // Get owner info
+    let to1d = get_to1d(dc.as_ref(), client_list).await.unwrap();
```

Comment:
Calling `.unwrap()` is a bad idea.

Reply:
The reason of doing this was that I would get a `Result<COSESign, Error>` instead of a `COSESign`. What should I do instead of using `.unwrap()` and why is it a bad idea?

Reply:
http://web.mit.edu/rust-lang_v1.25/arch/amd64_ubuntu1404/share/doc/rust/html/book/first-edition/error-handling.html#unwrapping-explained

Basically it's bad to just panic and exit here - you can `match` instead and handle any error

Reply:
Or you can use [the `?` operator](https://doc.rust-lang.org/edition-guide/rust-2018/error-handling-and-panics/the-question-mark-operator-for-easier-error-handling.html).

Reply:
`?` worked like a charm. Thank you both!